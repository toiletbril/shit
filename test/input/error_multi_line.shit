2 + 2

*

(2 + 2)

happiness
