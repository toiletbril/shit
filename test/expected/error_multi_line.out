shit: 7:1: Expected a binary operator, found 'happiness'.
     7 |  happiness
          ^~ Error happened here.
